### ISSUE TITLE
Create press kit

### ISSUE THREAD

**Issue by m52go** [bisq-network/growth#203]
Often, journalists and other people will cover Bisq and get something wrong: some element of its history, something about the DAO, etc.

Rarely someone will cover Bisq and get everything right—either because they took the time to talk to a contributor directly, or because they took the time to look through and read Bisq documentation.

It's not fair to expect a journalist trying to be first to break a story (or anyone else, really) to spend hours learning about Bisq to write an article on it. Sometimes it's a minor mistake, but often it's not.

There should be a press kit to condenses key information about Bisq into a simple package so that anyone can get the basics quickly.

Such a package should include, at a minimum:
- short history of Bisq
- abbreviated timeline of key events, releases, and release notes
- logo and other important assets (mark, font, colors, etc)
- gist of the DAO, how it works, etc
- gist of the security incident and what happened
- most active markets and approximate volumes (can probably just link to new bisq.markets when it's done)
- key media mentions, event presences, and/or videos that would be nice to show off (e.g. for people to mention or include somewhere in their article)

This will likely be another page on the website. Presentation doesn't need to be particularly fancy, but items should be presented cleanly so people can find exactly what they need and not get lost.

**Please speak up if you're interested in helping with this.** It will likely need to be a joint effort since most of these items are scattered. 

Relative to other bounties in this repository, it's not as high of a priority, but can still be completed in parallel with other tasks.

**Comment by pazza83** [comment]
I really like idea.

I would be happy to get involved in this. 

I would be happy to deliver this as a completed downloadable PDF with some nice graphics that could be added as to a press page on the website.

Also happy to complete complete as a website XD prototype that can then be added to the website.

**Comment by m52go** [comment]
Cool! I've seen some press pages delivered as a PDF for certain information. My opinion is that any kind of downloadable file is on the inconvenient side nowadays and it might be better to just put everything in a web page (might be better for SEO too? not sure).

Anyway would you like to get started with a rough outline for content and/or design?

**Comment by pazza83** [comment]
> Anyway would you like to get started with a rough outline for content and/or design?

Yes that is great. Happy to both PDF and website page. I will put the content together before the design and send it you for review before moving onto design.

**Comment by pazza83** [comment]
Do you have a brand pack / guidelines outlining:

- Fonts
- Typography style / hierarchy 
- Color palettes / themes
- Logos
- Brand elements
- Graphic elements
- Illustrations
- Iconography / symbols
- Web assets
- Print assets
- Event assets

Is there a repository for design files? For example previous designs for live events, GUI, website etc. Or files such as: PSD, AI, XD, IND, TTF.

Happy to from scratch, but if I do not have to it would take less time.

**Comment by m52go** [comment]
There is! 

https://github.com/bisq-network/bisq-design

See:
- `bisq-refresh.pdf` for colors
- `IBM-plex (bisq's official typeface).zip` file for fonts
- `bisq-logo+mark-0.9.0` folder for logos
- `bisq-campaings/2020.05` for some recent branding assets

**Comment by pazza83** [comment]
Thanks that is great 👍

**Comment by pazza83** [comment]
Hi @m52go 

Please see below my draft for the content. I would welcome any feedback.

I will make it look pretty in the design phase, the below is just the content that will be included for the XD and PDF designs.


---




# Media

Where possible Bisq is happy to help journalists, podcasters, and youtubers with their stories about Bitcoin, decentralized autonomous organizations, and peer to peer networks by putting them in contact with our contributors.

## Contact Info

Bisq is decentralized so the best way to reach out is via our various channels:

- Keybase
- Github
- Reddit
- Bisq Community
- Twitter
- Telegram

## Logos

Here are a few logos you can you use in your content.

- Add logos here
- Add logos here
- Add logos here

## Why are we called Bisq?

The name Bisq is the abbreviation of Bit (BI) and square (SQ)
This name was by the phrase Satoshi Square, a place place where people can bitcoin with other people. This represents the peer to peer aspects of Bisq and Bitcoin being it’s core building blocks.

## Founders

- Manfred Karrer https://github.com/ManfredKarrer
- Chris Beams - https://github.com/cbeams 


## Short history of Bisq

Here is a timeline showing important dates for Bisq:

- Feb 2014 Bank Run   https://bitcointalk.org/index.php?topic=462236  
- June 2010  Bitsquare   https://bitcointalk.org/index.php?topic=647457
- Aug 2014 Bitsquare website launces
- Dec 2014 Bitsquare v0.01 launces https://bisq.network/blog/bitsquare-now-in-alpha/
- Mar 2017 Bitsquare becomes Bisq https://bisq.network/blog/bitsquare-becomes-bisq/
- Oct 2017 Bisq Whitepaper  https://bitcointalk.org/index.php?topic=647457.0 
- Oct 2017 DAO Phae Zero  https://bitcointalk.org/index.php?topic=647457.0 
- Apr 2019 Bisq v1.0 launces  https://bisq.network/blog/dao-launch-third-anniversary/


## We’re a DAO. But what does that mean?

The DAO is an acronym for decentralized autonomous organization . 
Just as the Bisq exchange is decentralized and censorship-resistant, so is its governance mode. The Bisq DAO is one of the tools that achieves this.The Bisq DAO aims to create a self-contained economy where governance is guided by software but ultimately determined by the collective, subjective judgments of its community.

INCLUDE VIDEO https://www.youtube.com/watch?v=pNvOZlIDYEQ

More Bisq DAO resources:

- Introduction to the Bisq DAO — Learn how the DAO enables Bisq’s decentralized governance model
- User Guide: Bisq DAO for Traders — Use BSQ to get lower trading fees and compensate contributors
- User Guide: Bisq DAO for Contributors — Govern Bisq through the DAO by making proposals and voting
- Bisq DAO User Reference — Practical details on using the DAO, along with a brief technical overview
- Bisq DAO Technical Overview — Technical details of DAO functions and transactions
- Phase Zero -  https://docs.bisq.network/dao/phase-zero.html


## What makes us different?

What set apart Bisq from other exchnages is the follwing

- No Identity Verification
- Decentralized
- Safe
- Private
- Open
- Easy to Use
- It's Governance - The DAO

## Statistics

I will add Some nice graphics from Bisq market / coindance / GUI et


## Areas of expertise

Our contributors are experts in lots of different areas these include:

- Peer to peer networks
- Decentralized autonomous organization
- Privacy
- Anonymity
- Cryptography
- Bitcoin

## Bisq in the news

Some nice graphics similar to what i did on the growth slides

**Comment by m52go** [comment]
@pazza83 thanks -- I think content is the lion's share of what we should focus on for now anyway. Press kit, in my opinion, at this point, should be a simple page on the website without too much effort or fancy graphics. Just a way for people writing about Bisq to get what they need quickly and be accurate about whatever they're writing about.

I think what we have so far is enough to make a website pull request (PR), which we can use as a basis for further discussion. I'll make the PR in the next few days.

**Comment by pazza83** [comment]
Hi @m52go 

No problems. Would you like me to complete leave there then?

**Comment by m52go** [comment]
Yes -- I left the issue open since the larger item isn't yet delivered. We'll continue discussion on the pull request (separate thread).
